documentation/src/main/wiki/ProductionUsage.md

```diff
@@ -24,13 +24,21 @@ The *core* module contains 3 Knot.x verticle without any sample data. Here's how
       "preserved.headers": [
         "User-Agent",
         "X-Solr-Core-Key",
-        "X-Language-Code"
+        "X-Language-Code",
+        "X-Requested-With"
       ],
-      "dependencies" : {
-        "repository.address" : "template-repository",
+      "dependencies": {
+        "repository.address": "template-repository",
         "engine.address": "template-engine"
       }
-    }
+    },
+    "worker" : false,
```

Comment:
Remove those deployment properties from production usage as it might suggest that's the setting should be. Instead refer to the documentation where you explain possibilities. And here simply mention that each verticle can be configured/scaled using those properties